### PR TITLE
Create Sanity Action

### PR DESCRIPTION
**Pull request by nunnatsa**
Signed-off-by: Nahshon Unna-Tsameret <nunnatsa@redhat.com>

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: .github/workflows/pr-sanity.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
+# This is a basic workflow to help you get started with Actions
+
+name: Sanity Checks
+
+# Controls when the action will run.
+on:
+  pull_request:
+    branches: [ master ]
```
Comment (orenc1) [inline .github/workflows/pr-sanity.yaml:8]: what about cherry-pick pull requests on release branches?

Comment (nunnatsa) [reply .github/workflows/pr-sanity.yaml:8]: I almost sure that travis is not running there

**Pull request merged**